js/templates/modals/editListing/viewListingLinks.html

```diff
@@ -1,3 +1,6 @@
-<span class="viewListingLinks">
-  <a class="txU js-viewListing viewListingLink"><%= ob.polyT('editListing.viewListingLink') %></a>
-</span>
\ No newline at end of file
+<div class="gutterH <% if (ob.createMode) print('hide') %>">
+  <a class="txU js-viewListing clrTEmph1"><%= ob.polyT('editListing.viewListingInClientLink') %></a>
+  <a class="txU clrTEmph1" href="http://openbazaar.com/store/<%= ob.peerID %>/<%= ob.slug %>"><%= ob.polyT('editListing.viewListingOnWebLink') %><i class="ion-android-open padSm clrTEmph1"></i></a>
+  <span class="toolTip toolTipChatOpen txCtr" data-tip="<%= ob.polyT('editListing.viewOnWebDisclaimerTooltip') %>"><i class="ion-information-circled clrT2"></i></span>
+  </a>
```

Comment:
> You have a stray closing anchor here, and the tooltip isn't working.
Thanks for catching that extra closing anchor!

The tooltip was/is working just fine for me though... Did you test this on Windows? Just wondering what the difference was.

>I think the tooltip should be active when the link or the icon is hovered over. It should be something more like...

Agreed. 

Also, we just talked about it - so I'm removing the info icon since its redundant. @morebrownies : FYI: I removed the info icon since the tooltip now derives from the 'view on web' link and its redundant. Let me know if there's any issue with that.

Reply:
@morebrownies : example of what I changed it to: <img width="333" alt="screen shot 2018-06-12 at 5 16 33 pm" src="https://user-images.githubusercontent.com/392798/41318546-b5780950-6e66-11e8-973c-68b295110a1c.png">

Reply:
I like it @jashot7 👍